### DISCUSSION TITLE
Built-in VScode Terminal

### DISCUSSION THREAD

**Discussion by hamedg68** [vikejs/vike#740] [Discussion] [answered]
in last update **version 0.4.103** if I use built-in vscode terminal and run command _**yarn dev --host**_ VSCode crashes and closed

**Comment by brillout** [comment, marked as answer]
Is there anything that makes you believe it's related to VPS?

**Reply by hamedg68** [reply]
Version 0.4.102 works fine

**Reply by brillout** [reply]
It's probably due to https://github.com/brillout/vite-plugin-ssr/commit/c8e28a65245a7171667fba177d3c4b2982a577a7 then. What does your `yarn dev` command do?

**Reply by hamedg68** [reply]
```json
 "scripts": {
   "dev": "vite"
 }
```
I have to mention that I ran the`vue-full` example without any modification to ensure the problem is not with my code. Even I made test on other PCs.

**Reply by brillout** [reply]
Neat & thanks that's helpful. I've an idea of what's going on, I'll fix this. Closing in favor of https://github.com/brillout/vite-plugin-ssr/issues/742.
